notebooks/official/feature_store/sdk_feature_store.ipynb

```diff
@@ -0,0 +1,2418 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        labels={"status": "passed"}
</pre><p>NIT: prefer not to include labels if can not have a more meaningful label example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Took out the labels argument.